tests/tests/test_rvs/test_dynrvs.py

```diff
@@ -0,0 +1,32 @@
+import phoebe
+import numpy as np
+
+
+def test_dynrvs(verbose=False):
+    """
+    This test checks whether PHOEBE computes dynamical RVs without building
+    a mesh, and whether the computed RVs are close to the flux-weighted RVs.
```

Comment:
```suggestion
    This test checks whether PHOEBE computes dynamical RVs without building
    a mesh (as otherwise run_compute would fail with atm out-of-bounds errors), 
    and whether the computed RVs are close to the flux-weighted RVs.
```